NetCoreRestApi/DataLayer/UnitOfWorks/IUnitOfWork.cs

```diff
@@ -0,0 +1,12 @@
+﻿using DataLayer.Models;
+using DataLayer.Repositories;
+
+namespace DataLayer.UnitOfWorks
+{
+    public interface IUnitOfWork
+    {
+        IRepository<ProductModel> Products { get; }
+        IRepository<CategoryModel> Categories { get; }
```

Comment:
Remove these methods  and add all the same methods from IRepository interface

Reply:
done